.travis.yml

```diff
@@ -1,9 +1,9 @@
 ---
 language: node_js
 node_js:
-  - "4"
```

Comment:
This seems somewhat unrelated, and generally considered breaking (since we would no longer be testing Node 4).

Could you make this change in a separate PR and update the `package.json`'s `engines` field? That will help keep things together for the changelog...

Reply:
Open a pr with the change:
https://github.com/ember-cli/ember-exam/pull/148